num_repetitions: 1
debug_step: 1
max_fps: 30
Kp: 0 
Kd: 0 
compliance: 0.0
material: "metal"
horizon: 1000
dt: 0.001
tol: 0.000001
tol_rel: 0.000001
mu_prox: 0.0001
maxit: 100
warm_start: 1
contact_solver: "ADMM"
seed: 1234
add_damping: False
damping_factor: 0.0
admm_update_rule: "spectral"
max_patch_size: 4
patch_tolerance: 0.001
